google-cloud-spanner/acceptance/spanner/client/types/float32_test.rb

```diff
@@ -0,0 +1,228 @@
+# Copyright 2024 Google LLC
+#
```

Comment:
We may have to add params test for float32 as well and ensure it is handled in code.

Reply:
And if user does not specify the param type what would take precedence Float32 or 64 ?

Reply:
> We may have to add params test for float32 as well and ensure it is handled in code.

Missed this, thanks for spotting it. I added the tests for params in the `params/float32_test.rb` file.

Reply:
> And if user does not specify the param type what would take precedence Float32 or 64 ?

It defaults to float64 (there is already an existing test for that).